Fix some typos in data-registries.md and import.md (#2327)
iterative/dvc.org: content/docs/use-cases/data-registries.md

```

Besides downloading the data, importing saves the information about the

> dependency from the data source (registry repo) to the local project. This is

Reviewer: Hmm no, I realize this wasn't worded clearly, but we mean the dependency that the local project has to the data source i.e. the local project depends on the data source. Let me think... ⌛

Author: I see what you mean, maybe "dependency of the local project on the data source (registry repo)"

> dependency from the data source (registry repo) to the local project. This is

Reviewer: Let's do this for now:
```suggestion
dependency that the local project has on the data source (registry repo). This is
```
